python/paddle/tensor/manipulation.py

```diff
@@ -4345,11 +4345,18 @@ def reshape(x, shape, name=None):
 
         - 3. Given a 3-D tensor x with a shape [2, 4, 6], and the target shape is [-1, 0, 3, 2], the reshape operator will transform x into a 4-D tensor with shape [2, 4, 3, 2] and leaving x's data unchanged. In this case, besides -1, 0 means the actual dimension value is going to be copied from the corresponding dimension of x.
 
+    The following figure illustrates the first example -- a 3D tensor of shape [2, 4, 6] is transformed into a 2D tensor of shape [6, 8], during which the order and values of the elements in the tensor remain unchanged. The elements in the two subdiagrams correspond to each other, clearly demonstrating how the reshape API works.
+
+    .. image:: https://github.com/PaddlePaddle/docs/blob/develop/docs/images/api_legend/reshape.png
```

Comment:
```suggestion
    .. image:: https://githubraw.cdn.bcebos.com/PaddlePaddle/docs/develop/docs/images/api_legend/reshape.png
```
`https://githubraw.cdn.bcebos.com/PaddlePaddle/docs/develop/docs/images/api_legend/reshape.png
` 用这个路径吧，否则图片可能加载不出来，参考https://github.com/PaddlePaddle/Paddle/blob/78b038db01ca5ec3327539ece7e26bb6627243a6/python/paddle/distributed/communication/all_gather.py#L35-L38